graphql_compiler/compiler/expressions.py

```diff
@@ -204,6 +200,10 @@ def __init__(self, field_name):
         self.field_name = field_name
         self.validate()
 
+    def get_local_object_gremlin_name(self):
```

Comment:
Can this please be made a constant ? It seems static.

Reply:
It is a constant -- it is so constant, in fact, that you can't even get it without calling the function that hides it. I really don't want to pull it out of here, because classes that inherit from this one have no business reading that constant.

Reply:
If they shouldn't, then make it private here without exposing any function at all. But, this is fine too.

Reply:
Resolved offline.